modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc

```diff
@@ -10,9 +10,9 @@ For more information on Red Hat Technology Preview features, see https://access.
 Additional detail on how Red Hat provides support for bundled community dynamic plugins is available on the https://access.redhat.com/policy/developerhub-support-policy[Red Hat Developer Support Policy] page.
 ====
 
-There are 47 plugins.
+There are more than 45 plugins available in {product}. See the xref:rhdh-supported-plugins[Dynamic plugins support matrix] for details. 
```

Comment:
I'd suggest stating the actual number of available plugins rather than saying "more than 45", I know it takes a bit more maintenance but maybe some users want to know exactly how many. This will change again before GA as Nick has a PR adding 9 more plugins, so I think it will be 56.

Reply:
Yes, I was definitely trying to keep us from being wrong/out of date the moment they add another plugin, but I can change this back to the exact number.

Reply:
Plugins will only be added in each quarterly release, so it's probably a maintenance overhead that we can live with. Also in future releases, I think the plan might be to automate the generation of this table content, so we may not have to worry about it.